locales/en/plugin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Save": "Save",
```

Comment:
why do we have this file?
I think you were using `useTranslation` instead of `useKubevirtTranslation` somewhere